pkg/client/client.go

```diff
@@ -416,3 +461,27 @@ func (c *Client) indexes(tableName string) ([][]string, []string, error) {
 
 	return c.Query(query, args...)
 }
+
+func getDB(driver, connString, database string) (*sqlx.DB, error) {
+	var newConnString string
+
+	switch driver {
+	case drivers.MySQL:
+		newConnString = strings.Replace(connString, "/", fmt.Sprintf("/%s", database), 1)
+	default:
+		u, err := url.Parse(connString)
+		if err != nil {
+			return nil, err
+		}
+
+		u.Path = "/" + database
+		newConnString = u.String()
```

Comment:
This would be safer (I might have forgotten some)

```suggestion
	case drivers.PostgreSQL, drivers.Postgres:
		u, err := url.Parse(connString)
		if err != nil {
			return nil, err
		}

		u.Path = "/" + database
		newConnString = u.String()

	default:
		return nil, fmt.Errorf("unsupported driver %s", driver)
```

Reply:
I disagree with this, since the driver validation is done somewhere else and the MySQL URL parsing is the corner case here The default behavior is parsing the `DSN` with `url.Parse`.

Reply:
Ah OK then. I misunderstood